.build

```diff
@@ -1,4 +1,5 @@
 CC = gcc
```

Comment:
it's not at all necessary but I really don't want to be typing `ARCH=amd64 bldy //:kernel` everytime. contents of this file is overwritten if you specify a real env var like so `ARCH=risc-v bldy //:kernel` bldy will overwrite the ARCH set in .file